lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -0,0 +1,330 @@
+					}"
+					:style="{ left: range.percentValueFrom + '%', width: range.length + '%' }"
+				>
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+@import '../../../styles/settings/media-queries';
+@import '../../../styles/settings/colors/tokens';
+
+$progress-bar-height: 16px;
+$progress-bar-s-height: 8px;
+$progress-bar-xs-height: 4px;
+
+$progress-bar-border-radius: 8px;
+
+$progress-bar-layers: (
+	'primary': (
+		'default': $color-primary-text,
+		'ghost': $color-primary-data-ghost,
+		'medium': $color-primary-data-medium,
+		'weak': $color-primary-data-weak,
+		'neutral-weak': $color-neutral-data-weak,
+	),
+	'info': (
+		'default': $color-info-text,
+		'ghost': $color-info-data-ghost,
+		'medium': $color-info-data-medium,
+		'weak': $color-info-data-weak,
+		'neutral-weak': $color-neutral-data-weak,
+	),
+	'neutral': (
+		'default': $color-neutral-data,
+		'ghost': $color-neutral-data-ghost,
+		'medium': $color-neutral-data-medium,
+		'weak': $color-neutral-data-weak,
+		'neutral-weak': $color-neutral-data-weak,
+	),
+	'success': (
+		'default': $color-success-text,
+		'ghost': $color-success-data-ghost,
+		'medium': $color-success-data-medium,
+		'weak': $color-success-data-weak,
+		'neutral-weak': $color-neutral-data-weak,
+	),
+	'warning': (
+		'default': $color-warning-text,
+		'ghost': $color-warning-data-ghost,
+		'medium': $color-warning-data-medium,
+		'weak': $color-warning-data-weak,
+		'neutral-weak': $color-neutral-data-weak,
+	),
+	'fail': (
+		'default': $color-fail-text,
+		'ghost': $color-fail-data-ghost,
+		'medium': $color-fail-data-medium,
+		'weak': $color-fail-data-weak,
+		'neutral-weak': $color-neutral-data-weak,
+	),
+);
+
+.progressBar {
+	$self: &;
+
+	@each $color-name, $color-map in $progress-bar-layers {
+		&.-#{$color-name} {
+			#{$self}__result {
+				background: map-get($color-map, 'default');
+
+				&.-secondary {
+					background: map-get($color-map, 'ghost');
+				}
+			}
+
+			&.-schemeMedium {
+				#{$self}__result {
+					background: map-get($color-map, 'medium');
+
+					&.-secondary {
+						background: map-get($color-map, 'weak');
+					}
+				}
+			}
+
+			&.-schemeMediumNeutral {
+				#{$self}__result {
+					background: map-get($color-map, 'medium');
+
+					&.-secondary {
+						background: map-get($color-map, 'neutral-weak');
+					}
+				}
+			}
+		}
+	}
+
+	&-noRadius {
+		border-radius: 0;
+	}
+
+	&.-compact {
+		#{$self}__labelText {
+			@include headlineXS();
+		}
+		#{$self}__label {
+			margin-bottom: $space-xxxs;
+		}
+	}
+
+	&__bar {
+		background-color: $color-default-background;
+		border-radius: $progress-bar-border-radius;
+		box-shadow: inset 0 1px 4px $color-default-shadow-heavy;
+		height: $progress-bar-height;
+		overflow: hidden;
+
+		&.-small {
+			height: $progress-bar-s-height;
+		}
+
+		&.-xsmall {
+			border-radius: 0;
```

Comment:
where did we agree on disabling radius for this size?

Reply:
True, it's a mistake.